packages/radix-vue/src/shared/useActiveElement.ts

```diff
@@ -0,0 +1,12 @@
+export function useActiveElement(): Element | null {

```

Comment:
Since this is plain js function, and need not to rely on Vue's reactivity, let's not use the composable naming convention (`useXXX`).

Instead let's rename this function to be `getActiveElement` 😁

Reply:
Fair enough. I’ll update it sometime today. Thanks for the review